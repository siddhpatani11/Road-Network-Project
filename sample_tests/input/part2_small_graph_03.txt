15 74
0 5 51 G
0 7 41 R
0 9 30 G
0 11 43 B
0 12 1 G
0 14 49 G
1 0 8 B
1 2 2 R
1 4 40 R
1 6 48 R
1 7 33 B
1 10 38 G
2 1 19 B
2 3 17 G
2 4 7 G
2 5 17 G
2 7 32 B
2 9 47 G
2 11 41 R
2 12 46 G
2 13 50 B
3 1 15 G
3 5 3 G
4 1 8 G
4 3 39 R
4 5 29 R
4 7 13 G
5 3 9 R
5 4 31 R
5 8 19 B
6 1 42 R
6 8 7 R
6 11 36 B
6 14 21 R
7 2 17 G
7 4 44 G
7 5 31 R
7 12 23 G
7 13 12 B
8 1 29 R
8 4 19 G
8 5 36 B
8 6 28 B
8 7 14 G
8 10 20 G
8 13 28 R
9 0 47 R
9 5 20 R
9 6 9 G
9 7 2 G
9 8 3 B
9 11 41 R
9 14 21 B
10 0 12 G
10 1 8 R
10 4 44 R
10 7 27 R
11 0 20 B
11 7 11 R
11 8 9 G
11 10 2 G
12 1 8 B
12 2 41 B
12 6 6 B
12 7 29 R
12 8 46 B
12 10 20 R
13 3 9 B
13 6 2 B
13 7 2 G
14 1 44 B
14 3 40 G
14 7 29 B
14 8 6 G
