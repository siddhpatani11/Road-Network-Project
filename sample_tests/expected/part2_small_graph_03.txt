0 - 0 G 9 G 11 G 28 G 18 G 28 G 7 G 28 G 14 G 30 G 34 B 43 G 1 G 40 G 28 
